src/main/java/atrium/models/Challenge.java

```diff
@@ -41,6 +51,7 @@ public String toString() {
         sb.append(", label='").append(label).append('\'');
         sb.append(", options='").append(Arrays.toString(options)).append('\'');
         sb.append(", type='").append(type).append('\'');
+        sb.append(", value='").append(value).append('\'');
```

Comment:
I would reach for a real Param builder since values that include quotes would break this code.